docs/source/api/model/transform/conditioning.rst

```diff
@@ -0,0 +1,19 @@
+Model Conditioning
```

Comment:
Can you add `fgraph_from_model`, `model_from_fgraph` and `clone_model` to the docs as well?

Reply:
fixed in https://github.com/pymc-devs/pymc/pull/6879/commits/aaa91a8e6d14f6a37f5b6ab5ae95043fff44f25e